card/mod/03_machines/lib/javascript/wagn.js.coffee

```diff
@@ -253,6 +253,11 @@ $(window).ready ->
 
   $('body').on 'change', '.edit-type-field', ->
     $(this).closest('form').submit()
+    
+  $('body').on 'change', '.submit-select-field', ->
+    field = $(this).closest('.card-editor').find('.card-content')
+    field.val( $(this).val() )
+    $(this).closest('form').submit()
```

Comment:
no, that was part of an outdated 'follow' interface (list of text fields without submit buttons to update and create follow rules). I'll remove it.